src/libs/dal/Services/FileSystemHistoryItemService.cs

```diff
@@ -41,11 +42,14 @@ public IEnumerable<FileSystemHistoryItem> Find(FileSystemHistoryItemFilter filte
     {
         var query = (from fsi in this.Context.FileSystemHistoryItems
                      join si in this.Context.ServerItems on fsi.FileSystemItem!.ServerItemServiceNowKey equals si.ServiceNowKey
-                     join tenant in this.Context.Tenants on si.TenantId equals tenant.Id
+                     join org in this.Context.Organizations on si.OrganizationId equals org.Id
```

Comment:
Need to join on organization because many server items do not directly link to a tenant